src/main/java/com/linkedin/replica/wall/commands/impl/EditPostCommand.java

```diff
@@ -1,5 +1,6 @@
 package com.linkedin.replica.wall.commands.impl;
```

Comment:
Edit command should be reimplemented as we discussed. It should only update fields with passed values in args. it will need Arango query
@loayelzobeidy